pkg/sql/sqlbase/testutils.go

```diff
@@ -103,25 +105,49 @@ func RandDatum(rng *rand.Rand, typ ColumnType_Kind, null bool) parser.Datum {
 		return parser.NewDBytes(parser.DBytes(p))
 	case ColumnType_TIMESTAMPTZ:
 		return &parser.DTimestampTZ{Time: time.Unix(rng.Int63n(1000000), rng.Int63n(1000000))}
+	case ColumnType_COLLATEDSTRING:
+		if typ.Locale == nil {
+			panic("locale is required for COLLATEDSTRING")
+		}
+		// Generate a random Unicode string.
+		var buf bytes.Buffer
+		for i := 0; i < 10; i++ {
+			var r rune
+			for {
+				r = rune(rng.Intn(unicode.MaxRune + 1))
+				if !unicode.Is(unicode.C, r) {
+					break
+				}
```

Comment:
could you use of these http://stackoverflow.com/questions/22892120/how-to-generate-a-random-string-of-a-fixed-length-in-golang

?